benchmarks/distributed/rpc/parameter_server/trainers/RpcTrainerBase.py

```diff
@@ -0,0 +1,54 @@
+import torch
+import torch.distributed.rpc as rpc
+
+from .TrainerBase import TrainerBase
+
+
+class RpcTrainerBase(TrainerBase):
+
+    RPC_FUT_METRIC = "rpc_fut_metric"
+    RPC_FUT = "rpc_fut"
+    RPC_METRIC = "rpc_metric"
+    RPC = "rpc"
+
+    def __init__(self, rank):
+        super().__init__(rank)
+
+    def send_async_request(self, key, rref, server_method, *args, cuda=True):
+        self.record_rpc_fut_start(key, f"{self.RPC_FUT}_{server_method.__name__}", cuda)
+        fut = rpc.rpc_async(
```

Comment:
there are RRef helpers to simplify this. I.e.:

```python
rref.rpc_async().server_method(*args)
```

Reply:
okay will update. I might end up moving this method.

Reply:
removed the method.